fast64_internal/sm64/sm64_geolayout_writer.py

```diff
@@ -978,7 +977,7 @@ def exportGeolayoutObjectBinary(
     camera,
 ):
     geolayoutGraph, fModel = convertObjectToGeolayout(
-        obj, convertTransformMatrix, camera, obj.name, None, None, DLFormat.Static, True
+        obj, convertTransformMatrix, True, obj.name, None, None, DLFormat.Static, True
```

Comment:
You should remove the "camera" argument from the functions if it is not used.

Reply:
that is what I did?

Reply:
yeah remove the `camera` from the function `exportGeolayoutObjectBinary` and others that do not actually use the argument.